infra/permissions/main.tf

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "4.48.0"
+    }
+  }
+}
```

Comment:
Where this the tf state going be stored? Typically it's best to put it in a bucket, e.g.,

```
terraform {
  backend "gcs" {
    bucket = "gcping-tf-state"
  }
}
```

Reply:
Since this is only for local development, the state is only stored locally.